frontend/src/metabase/parameters/components/ParameterValueWidget.css

```diff
@@ -37,28 +37,34 @@
   border-color: var(--color-brand);
 }
 
+:local(.parameter) input {
+  flex-basis: 142px;
+  flex-grow: 1;
+  border-radius: 0;
+}
+
 :local(.parameter.noPopover) input {
   width: 100%;
   font-size: 1em;
   font-weight: 600;
   border: none;
   background: none;
-}
 
-:local(.parameter.noPopover.isEditing) input {
-  width: 142px;
+  @media screen and (min-width: 440px) {
+    & {
+      padding: 0;
+    }
+  }
```

Comment:
I don't understand why this is only applied on `min-width: 440px` - [other inputs in the sidebar don't have this](https://github.com/metabase/metabase/assets/6830683/ed661e2e-cbfa-4ed7-b742-5edf23ebf4d4).


```suggestion
  padding: 0;
```

Reply:
It looks weird without `padding: 0` - I think it should be the default value without any media query.

<img width="361" alt="Screenshot 2023-11-23 at 12 33 15" src="https://github.com/metabase/metabase/assets/8542534/72de46cc-c564-4ab9-b8df-a83b5f1d3619">

Reply:
I have not. But given that `ParameterValueWidget` is used in `TagEditorParam`, which I haven't tested (and frankly I think it's outside the scope of this PR), I will leave this part untouched.